mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -5428,7 +5579,790 @@
+													"    pm.expect(tv4.missing.length).to.equal(0, \"Missing schemas: \" + JSON.stringify(tv4.missing));",
+													"});",
+													"",
+													"pm.test(\"Errors array is as expected\", function () {",
+													"    pm.expect(response.errors).to.be.an('array').that.is.not.empty;",
+													"    pm.expect(response.errors.length).to.eql(1);",
+													"    pm.expect(response.errors[0].title).to.equal(\"Invalid customerId\");",
+													"    pm.expect(response.errors[0].detail).to.equal(\"customerId must not be empty\");",
+													"});"
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"method": "PUT",
+										"header": [
+											{
+												"key": "x-okapi-tenant",
+												"type": "text",
+												"value": "{{xokapitenant}}"
+											},
+											{
+												"key": "x-okapi-token",
+												"type": "text",
+												"value": "{{xokapitoken}}"
+											},
+											{
+												"key": "Content-Type",
+												"type": "text",
+												"value": "application/vnd.api+json"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": "{\n  \"data\": {\n    \"type\": \"kbCredentials\",\n    \"attributes\": {\n      \"name\": \"University of Illinois\",\n      \"apiKey\": \"{{rm.apiKey}}\",\n      \"url\": \"{{rm.apiUrl}}\",\n      \"customerId\": \"\"\n    }\n  }\n}\n",
+											"options": {
+												"raw": {
+													"language": "json"
+												}
+											}
+										},
+										"url": {
+											"raw": "{{protocol}}://{{url}}:{{okapiport}}/eholdings/kb-credentials/{{kb-credentials-id1}}",
+											"protocol": "{{protocol}}",
+											"host": [
+												"{{url}}"
+											],
+											"port": "{{okapiport}}",
+											"path": [
+												"eholdings",
+												"kb-credentials",
+												"{{kb-credentials-id1}}"
+											]
+										}
+									},
+									"response": []
+								}
+							],
+							"protocolProfileBehavior": {},
+							"_postman_isSubFolder": true
+						}
+					],
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "90e92342-38a1-4da8-b854-a0016eca3891",
+								"type": "text/javascript",
+								"exec": [
+									""
+								]
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "b6c69a3e-2754-430c-b9b9-944485d72884",
+								"type": "text/javascript",
+								"exec": [
+									""
+								]
+							}
+						}
+					],
+					"protocolProfileBehavior": {},
+					"_postman_isSubFolder": true
+				},
+				{
+					"name": "PATCH kb-credentials",
+					"item": [
+						{
+							"name": "Positive",
+							"item": [
+								{
+									"name": "PUT credentials",
```

Comment:
rename to PATCH and all other tests in "negative case"

Reply:
fixed